content/exchange/artifacts/Windows.ETW.ScreenshotTaken.yaml

```diff
@@ -0,0 +1,62 @@
+name: Windows.ETW.ScreenshotTaken
+author: Zane Gittins
+description: |
+   This artifact detects screen captures by correlating events from the Microsoft-Windows-Win32k ETW provider. This can be useful for detecting remote access trojans, infostealers, and data exfiltration. Tested against Sliver, Meterpreter, and Empire. This will also trigger on legitimate tools such as ZoomIt, Greenshot, MsTeams, etc. which can be excluded on a case by case basis via the ProcessExceptionsRegex parameter.
+
+# Can be CLIENT, CLIENT_EVENT, SERVER, SERVER_EVENT or NOTEBOOK
+type: CLIENT_EVENT
+
+parameters:
+  - name: ProcessExceptionsRegex
+    description: Except these processes.
+    type: string
+    default: "Explorer.exe"
+  - name: ScreenshotMaxRows
+    description: Maximum number of screenshot events to store in internal fifo queue for correlation.
+    type: int
+    default: 500
+  - name: ScreenshotMaxAge
+    description: Maximum age in seconds to retain screenshot events to store in internal fifo queue for correlation.
+    type: int
+    default: 60
+sources:
+  - precondition:
+      SELECT OS From info() where OS = 'windows'
+
+    query: |
+       // 1=WindowUpdate
+       LET WindowUpdateEvents = SELECT *, System.ProcessID AS FirstPid
+         FROM watch_etw(guid="{8c416c79-d49b-4f01-a467-e56d3aa8234c}",
+                        level=4,
+                        any=4096)
+         WHERE System.ID = 1
+          AND EventData.Hwnd = "0x0"
+               AND (EventData.Type = "2147483654" OR EventData.Type = "2147483655")
+       // 33=GdiSysMemToken OR 52=PhysicalSurfCreate
+       LET ScreenshotEvents = SELECT *
+         FROM watch_etw(guid="{8c416c79-d49b-4f01-a467-e56d3aa8234c}")
```

Comment:
could you please also add the description then it is easier to see in the etw profiles